11-change-ppp-mtu-debug.sh

```diff
@@ -74,14 +75,14 @@ while true; do
           if [ $pmtucorrect -eq 1 ] && [ $emtucorrect -eq 1 ]; then
             # Config files are all correct
             echo Config files are now all set correctly
-            runningmtu=$(ip link list | grep $pinterface | grep $(($PTARGET-8)))
-            if [ ! -z $runningmtu ]; then
+            runningmtu=$(ip link list | grep $pinterface | grep $(($PTARGET)))
```

Comment:
get rid of the -8 check here and look for if it doesnt have the correct value instead of it having the specific wrong value